LAB1.html

```diff
@@ -0,0 +1,200 @@
id="90FC742E" data-testid="output_8" style="width: 892px;"><div class="figureElement"><img class="figureImage figureContainingNode" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA0gAAAJ2CAYAAABl4KKyAAAgAElEQVR4AezBf4ydBaHu+69vS6f1EMKPkEKi9F1F75PYQO8QUCz7OoukmTEk5Hj+GCRR6Uw04I7UpjHZ/EqYtXCKmmiMaQQlMZ1SqBIwyI4B1oRr17DJ7BwDUYqc+qSYtUyNQZCCDXdsa2e8ZyaHnVlY6JTW0pl5Pp/i7xERERERETGjICIiIiIiImYURERERERExIyCiIiIiIiImFEQERERERERMwoiIiIiIiJiRkFERERERETMKIiIiIiIiIgZBRERERERETGjICIiIiIiImYURERERERExIyCiIiIiIiImFEQERERERERMwoiIiIiIiJiRkFERERERETMKIiIiIiIiIgZBRERERERETGjICIiIiIiImYURERERERExIyCiIiIiIiImFEQERERERERMwoiIiIiIiJiRkFERERERETMKIiIiIiIiIgZBRERERERETGjICIiIiIiImYURERERERExIyCiIiIiIiImFEQERERERERMwoiIiIiIiJiRkFERERERETMKIiIiIiIiIgZBRERERERETGjICIiIiIiImYURERERERExIyCiIiIiIiImFEQERERERERMwoiIiIiIiJiRkFERERERETMKIiIiIiIiIgZBcHU1BTj4+M8/vjjjI+PMzU1xYl49dVXeeqpp9i/fz9vNzk5ycTEBBMTE0xMTDAxMcHExAQTExMcOnSIiIiIiIh4/xQsci+88ALr169ncHCQzZs3Mzg4yPr169mzZw/v1aZNm/jKV77Cb3/7W96uXq/T3d1Nd3c33d3ddHd3093dTXd3N7fddhsREREREfH+KVjEDhw4wE033cTBgwe577772L17N1u3buXgwYPceOONvPnmmxyvH/7whzz33HO8k3379rFs2TL6+/vp7++nv7+f/v5++vv7+cQnPkFERERERLx/ChaxBx54gNdee40777yTnp4eurq66O3t5a677uKVV15hx44dHI89e/bwve99j3POOYd38uyzz3LllVcyPDzM8PAww8PDDA8PMzw8zGc/+1kiIiIiIuL9U7CINRoNli9fTm9vL7OtX7+e5cuX8+STTzJXhw4dYvPmzVx22WVcc801HM2+ffs4fPgwH/nIR4iIiIiIiNNPwSI1NTXF3r17ufTSSymKgre77LLL2Lt3L1NTU8zFt771Lf785z/zne98h3eyZ88epnV3d/Pss8/ywAMP8NOf/pQ9e/YQERERERHvv4JFamJigsnJSc4991yO5qyzzmJycpJDhw5xLLt27eLBBx9kaGiIlStX8k52797NtC1btvC5z32Or3/969x+++185jOf4ZZbbmFycpKIiIiIiHj/FCxSr7/+OtOWLVvG0SxZsoRp7Xabd7N//37uuOMOPv3pT3PttdfybtrtNtM+9rGP8fOf/5zf/OY3/OQnP+HSSy/lZz/7Gd/85jeZiy984QtIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlIQhKSkIQkJCEJSUhCEpKQhCQkIQlJSEISkpCEJCQhCUlI4gtf+AJxYgoWqeXLlzMXRVHwbm677TaWLFlCvV7nWG644Qa+/e1vc++99/LRj36UM844g+7ubrZt28Z5553Hzp07OXDgAMfyy1/+EtvYxja2sY1tbGMb29jGNraxjW1sYxvb2MY2trGNbWxjG9vYxja2sY1tb
```

Comment:
Title, legend, axis labels - plots should be self-contained so that I could look at only them and understand what they are